File: Java Codewars/8kyu/Beginner - Reduce but Grow.java
/*
Description

Given a non-empty array of integers, return the result of multiplying the values together in order. 

Example:
[1, 2, 3, 4] => 1 * 2 * 3 * 4 = 24
*/

import java.util.*;

public class Kata {

  public static int grow(int[] x) {
    return Arrays.stream(x).reduce(1, (a, b) -> a * b);
  }
}

File: C# Codewars/8kyu/Beginner - Reduce but Grow.cs
/*
Description

Given a non-empty array of integers, return the result of multiplying the values together in order. 

Example:
[1, 2, 3, 4] => 1 * 2 * 3 * 4 = 24
*/

using System;
using System.Linq;

public class Kata
{
    public static int Grow(int[] x)
    {
        return x.Aggregate((a, b) => a * b);
    }
}